test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2071,6 +2071,35 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
 
     #endregion charset encoding tests
 
+    Context 'Invoke-RestMethod ResponseHeadersVariable Tests' {
+        It "Verifies Invoke-RestMethod supports -ResponseHeadersVariable" {
+            $uri = Get-WebListenerUrl -Test '/'
+            $response = Invoke-RestMethod -Uri $uri -ResponseHeadersVariable 'headers'
+
+            $headers | Should Not BeNullOrEmpty
+            $headers.'Content-Type' | Should Be 'text/html; charset=utf-8'
+            $headers.Server | Should Be 'Kestrel'
+        }
+        It "Verifies Invoke-RestMethod supports -HV alias" {
```

Comment:
We never test parameter aliases. If we want test parameter aliases we must create one complex test. I think we should remove the test.

Reply:
Fixed.